src/signals/incident-management/containers/IncidentDetail/components/LocationPreview/index.js

```diff
@@ -4,28 +4,35 @@ import PropTypes from 'prop-types';
 import { locationType } from 'shared/types';
 import { markerIcon } from 'shared/services/configuration/map-markers';
 import styled from 'styled-components';
+import Button from 'components/Button';
+import { themeSpacing } from '@datapunt/asc-ui';
 import MapDetail from '../MapDetail';
 
-import './style.scss';
-
 const StyledMap = styled(MapDetail)`
   width: 100%;
   height: 450px;
 `;
 
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const StyledButton = styled(Button)`
+  position: absolute;
+  margin: 0;
+  top: ${themeSpacing(3)};
+  left: ${themeSpacing(3)};
+  z-index: 500;
```

Comment:
Previous spacing was 10px; does the change still align the button correctly?

Reply:
It looks ok, and by this we align the controls to the 4point grid